radixdlt/src/main/java/com/radixdlt/consensus/LedgerHeader.java

```diff
@@ -96,9 +101,19 @@ public static LedgerHeader create(
 		View view,
 		AccumulatorState accumulatorState,
 		long timestamp,
-		boolean isEndOfEpoch
+		BFTValidatorSet validatorSet
```

Comment:
Perhaps it worth to add similar create() method but without last parameter. There are a number of places across whole PR where this parameter is set to 'null'. Adding dedicated method will improve code maintainability.